### ISSUE TITLE
'NoneType' object has no attribute 'path' in Postgresql.__del__

### ISSUE THREAD

**Issue by nbonnotte** [tk0miya/testing.postgresql#3]
Great package! It works very well.

I just wanted to let you know: at the end of my tests, even the successful ones, I always get the message:
```
Exception AttributeError: "'NoneType' object has no attribute 'path'" in <bound method Postgresql.__del__ of <testing.postgresql.Postgresql object at 0x7f656550a4d0>> ignored
```
Am I doing something wrong?

**Comment by tk0miya** [comment]
Thank you for reporting. but I could not reproduce the problem.

Probably, it caused by gabage collection mechanism.
I try to fix the problem with the commit above.
Could you try it and send feedback please?

**Comment by nbonnotte** [comment]
Yep, that solved it. Thanks!

**Comment by tk0miya** [comment]
Thanks. I released v1.2.1 including this fix :-)
